src/vehicleProto.cpp

```diff
@@ -125,4 +128,12 @@ void CurrentVehicle::restartTinyBms() {
     
 }
 
+String computeHorizontalSpeed() {
+    if (ecu.getOn())
+                // 9.7 : 1 gear ratio -- 21" diameter wheels -- 39370.1 inches per kilometer
+                return String((float)ecu.getRPM() / 9.7 * 60 * 3.14 * 21 / 39370.1);
```

Comment:
I'd recommend wrapping every binary operation in brackets to be explicit about order of operations, it makes it more readable. Not required though.